kotlin/README.md

```
@@ -122,7 +122,7 @@ The following credential formats are supported for sharing:
     - [authenticateVerifier](#authenticateverifier)
     - [constructUnsignedVPToken](#constructUnsignedVPToken)
     - [shareVerifiablePresentation](#shareverifiablepresentation)
-    - [sendErrorToVerifier](#senderrortoverifier)
+    - [sendErrorResponseToVerifier](#senderrorresponsetoverifier)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)